NEWS.md

```diff
@@ -1,5 +1,9 @@
 # xportr 0.4.0
 
+## xportr development version
+
+* New argument in `xportr_write()` allows users to specify the maximum file size (in GB) of their exported xpt files. (#268)
```

Comment:
shouldn't the DESCRIPTION file version update to 0.5.0 and then include this new feature under a new section: 

```
# xportr 0.5.0

## New Features

```
Especially if this is the only update before you'll send back to CRAN.

Reply:
umm..yeah that should go above 0.4.0!! nice catch  we should probably just change to 0.5.0 and add skip vbump in the PR title.  @elimillera WDYT

Reply:
I agree. This adds new functionality so updating the minor version would be the way I would go

Reply:
Thanks, updated.